.github/workflows/conformance.yaml

```diff
@@ -93,6 +95,8 @@ jobs:
             version: v1.29.4
           - name: v1.30
             version: v1.30.0
+        upgrade-from:
+          - "3.2.6"
```

Comment:
Can we dynamically search the latest published chart on `main`, and pin the latest patch version on release branches?
- on the `main` branch, we can use helm command to search and grep the latest version
- on release branches, we can grep `version` from `charts/kyverno/Chart.yaml`.

Reply:
Looks reasonable. Will try to implement it.

Reply:
@realshuting I have updated it to trigger on schedule as well as on a pull request to a release branch.

Reply:
We can't do it on a `v*` tag because a race condition cold happen and the helm chart gets releases first. So it upgrades from itself as it will be the latest version.

> on release branches, we can grep version from charts/kyverno/Chart.yaml

This is not helpful? As we would not have the current release version. We still want the previous one. I made the same as schedule which is to get the latest helm chart.